lib/src/entrypoint.dart

```diff
@@ -111,15 +111,16 @@ class Entrypoint {
             path, {
             expectedName,
             required withPubspecOverrides,
-          }) =>
-              pubspecsMet[p.canonicalize(path)] ??
-              Pubspec.load(
-                path,
-                cache.sources,
-                expectedName: expectedName,
-                allowOverridesFile: withPubspecOverrides,
-                containingDescription: RootDescription(path),
-              ),
+          }) {
```

Comment:
is this just reformatting?

Reply:
Yeah - I had some debug prints in here I think. I'll revert this part.